### DISCUSSION TITLE
Keyframes Infinite Loop

### DISCUSSION THREAD

**Discussion by Nikita0x** [software-mansion/react-native-reanimated#5354] [Q&A]
How to make keyframes be running infinitely?????

**Comment by MatiPl01** [comment]
Hi!

If you still have questions, could you precise what do you mean by keyframes? Can you explain what you are trying to achieve? Do you use `useFrameCallback` hook or something else?

**Reply by Karthik-B-06** [reply]
Something like this https://markodenic.com/css-pulse-animation/ ?

**Reply by MatiPl01** [reply]
This is CSS, which will be supported in reanimated 4 soon but is not supported in the stable (reanimated v3) release.
